addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss

```diff
@@ -45,7 +45,6 @@
     position: absolute;
     z-index: 10;
     padding: 0;
-    border: $border-width solid var(--o-table-ui-bg, #{$border-color});
```

Comment:
The border wasn't visible? Doesn't this change the design?

Reply:
It was almost invisible as its container had no height or width (depending if the UI was for column or row).

See the UI for columns with the border:
![](https://i.imgur.com/EcSFNpv.png)
Without the border:
![](https://i.imgur.com/EJZwrWK.png)

There is an offset of 1px to the right with the border.

That border was probably intended for a prior version of the code.